nikola/plugin_categories.py

```diff
@@ -250,6 +250,7 @@ class PageCompiler(BasePlugin):
     friendly_name = ''
     demote_headers = False
     supports_onefile = True
+    use_dep_file = False  # If set to true, the .dep file is always written and added as a target
```

Comment:
Well, some compilers might use `.dep` files for something else. Actually, the WordPress compiler does: it uses the `.dep` file to store a JSON-encoded file which not only contains file dependencies, but potentially other dependencies. (I added that feature when `.dep` files were completely handled by the compilers themselves.) I don't know if any other plugin not shipped with Nikola directly uses `.dep` like this as well.